molo/surveys/forms.py

```diff
@@ -63,6 +110,28 @@ def create_datetime_field(self, field, options):
     def create_positive_number_field(self, field, options):
         return forms.DecimalField(min_value=0, **options)
 
+    def create_dropdown_field(self, field, options):
```

Comment:
do we need these here? Does this not already happen in wagtail surveys?

Reply:
had to override to add `return CharacterCountChoiceField(**options)`